loom/graphics/gfxVectorGraphics.cpp

```diff
@@ -170,7 +170,6 @@ void VectorGraphics::endFill() {
 void VectorGraphics::moveTo(float x, float y) {
     auto path = getPath();
     path->moveTo(x, y);
-    inflateBounds(Loom2D::Rectangle(x, y, 0, 0));
```

Comment:
I originally added this when doing the bounds, but it's not necessary. Move shoudn't inflate. Making it inflate and then removing some internal move caused the bug in the first place.